umh.docs.umh.app/content/en/docs/architecture/microservices/grafana-plugins/umh-datasource-v2.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Umh Datasource V2"
+content_type: grafana-plugin
+description: |
+    This page contains the technical documentation of the umh-datasource-v2 plugin,
+    which allows for easy data extraction from factoryinsight.
+weight: 1000
+---
+
+<!-- overview -->
+
+The plugin, **umh-datasource-v2**, is a Grafana data source plugin that allows you to fetch
+resources from a database and build queries for your dashboard.
+
+<!-- body -->
+
+## {{% heading "howitworks" %}}
+
+1. When creating a new panel, select **umh-datasource-v2** from the Data source drop-down menu. It will then fetch the resources
+   from the database. The loading time may depend on your internet speed.
+
+   ![selectingDatasource](/images/grafana-plugins/grafanaPluginsSelectingV2.png?width=80%)
+
+2. Select the resources in the cascade menu to build your query. **DefaultArea** and **DefaultProductionLine** are placeholders
+   for the future implementation of the new data model.
+
+   ![selectingDatasource](/images/grafana-plugins/grafanaPluginsSelectingWorkCell.png?width=80%)
+
+3. Only the available values for the specified work cell will be fetched from the database. You can then select which data value you want to query.
+
+   ![selectingDatasource](/images/grafana-plugins/grafanaPluginsSelectingValue.png?width=80%)
+
+4. Next you can specify how to transform the data, depending on what value you selected.
+   Below you can find the options to transform the queried data:
+
+   - **Time bucket**: lets you group data in a time bucket
+   - **Aggregates**: common statistical aggregations (maximum, minimum, sum or count)
```

Comment:
Please specify the available options here. Link to the timescaleDB documentation for each of them for the exact behaviour.

Reply:
Please list them out.
Average --> Calculates the average for each time bucket. See also https://docs.timescale.com/api/latest/hyperfunctions/statistical-and-regression-analysis/stats_agg-one-variable#average
...